docs/root/configuration/listener_filters/proxy_protocol_filter.rst

```diff
@@ -0,0 +1,13 @@
+.. _config_listener_filters_proxy_protocol:
```

Comment:
love the new docs. Can we also have a brief mention of this functionality in https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/listeners?

Reply:
Done.